### DISCUSSION TITLE
Configure `sass_email_location` at Mailer level in Rails

### DISCUSSION THREAD

**Discussion by ocarreterom** [bootstrap-email/bootstrap-email#170] [Help] [answered]
Is there a way to set up the `sass_email_location` in different Mailers? I have a project where 2 emails are sent with different style.

**Comment by stuyam** [comment, marked as answer]
There isn't a away to do this on a per mailer basis using the build in `bootstrap_mail` method. However the `bootstrap_mail` method is very simple and you could use it manually with the `BootstrapEmail::Compiler` class and pass in manual options for configuring the sass_email_location: https://github.com/bootstrap-email/bootstrap-email/blob/master/lib/bootstrap-email/rails/action_mailer.rb#L8-L9
